chainconsumer/plotter.py

```diff
@@ -910,8 +911,11 @@ def _plot_contour(self, ax, chain, px, py, truth=None, color_extents=None):  # p
             y_centers = np.linspace(y_bins.min(), y_bins.max(), nn)
             xx, yy = meshgrid(x_centers, y_centers, indexing='ij')
             coords = np.vstack((xx.flatten(), yy.flatten())).T
-            data = np.vstack((x, y)).T
-            hist = MegKDE(data, w, kde).evaluate(coords).reshape((nn, nn))
+            # data = np.vstack((x, y)).T
+            # hist = MegKDE(data, w, kde).evaluate(coords).reshape((nn, nn))
+            hist_kde, (x_centers_kde, y_centers_kde) = fastKDE.pdf(x,y)
```

Comment:
So here we use all the defaults, which might be fine. But would be good (for me) to play around with various things, such as the density of the axes (`numPointsPerSigma`), the potential impact of using `doApproximateECF`. 

I also need to think about backwards compatibility, in the sense that some people are used to specifying bandwidths. This implementation is nicer in that its automatic, but it might be useful if it was still controllable.

Reply:
Agreed, I'll try to understand that.